vite.config.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  server: {
+    open: 'none'
```

Comment:
just looking at the docs, should this not be `false`?

Reply:
Hmm now I wonder if there are two ways, or maybe the vite version they use is old? This seemed to work 🤷